### PR TITLE
Use the robots meta tag

### PR DESCRIPTION
**Pull request by ycanardeau**
Inspired by [MediaWiki](https://www.mediawiki.org/wiki/MediaWiki).

### CODE REVIEW

File: VocaDbWeb/Views/Album/Details.cshtml [hunk 1]
```diff
@@ -16,6 +16,8 @@
 @{
 	ViewBag.Parents = new[] { Html.ActionLink(ViewRes.SharedStrings.Albums, "Index") };
 	var url = PageProperties.CanonicalUrl;
+
+	PageProperties.Robots = Model.Deleted ? "noindex,follow" : string.Empty;
```
Comment (riipah) [inline VocaDbWeb/Views/Album/Details.cshtml:20]: Can you add a named constant for these?

**Comment by riipah** [comment]
Please add the named constant for the "robots" values at least (I think it's the same everywhere). And consider moving them to controller instead of view, if possible.

File: VocaDbWeb/Views/Album/Details.cshtml (hunk 1 above)
Comment (ycanardeau) [reply VocaDbWeb/Views/Album/Details.cshtml:20]: What would you name those constants? We have two different values, `noindex,follow` and `noindex,nofollow`. And where would you place them?

**Comment by ycanardeau** [comment]
Thanks for your review! I considered moving them to controller. but some methods like `UserController.Login` and `UserController.Create` have multiple overloads, so I think it's better not to do that.

File: VocaDbWeb/Views/Album/Details.cshtml (hunk 1 above)
Comment (riipah) [reply VocaDbWeb/Views/Album/Details.cshtml:20]: Maybe you could place them in the PagePropertiesData class? The name could be just ```Robots_NoIndex_Follow``` (I think it's still better to have named constant, even if the constant includes the whole string)

**Comment by ycanardeau** [comment]
Added named constants.

**Comment by riipah** [comment]
Thanks, it's ok now.

**Pull request merged**